.github/workflows/build-starter-templates.yml

```diff
@@ -54,20 +54,24 @@ jobs:
         id: report-errors
         if: failure()
         run: |
-          ISSUE_TITLE="Starter template '${{ matrix.template }}' failed to build with ${{ matrix.package-manager }}"
+          ISSUE_TITLE="Starter template '${{ matrix.template }}' failed to build with ${{ matrix.package-manager }} using Node ${{ matrix.node-version }}"
           RUN_URL="https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          ISSUE_BODY=$(cat <<EOF
+          The **${{ matrix.template }}** starter template failed to build with **${{ matrix.package-manager }}** using Node **${{ matrix.node-version }}**.
+
+          Check the full workflow logs for more details: [View Workflow Run]($RUN_URL)
```

Comment:
Possible to make this either, 

A. Go to the actual run 
B. Add the error output into the issue 🙏

Its very difficult to find the run when theres 30+ runs with the same names (not exactly)

Reply:
As per my conversation with Jack, getting the runners job ID is difficult when dealing with Matrices so for now we will continue to just link to the action.